src/bootstrap/format.rs

```diff
@@ -159,8 +159,14 @@ pub fn format(build: &Builder<'_>, check: bool, paths: &[PathBuf]) {
             if !check && paths.is_empty() {
                 match get_modified_rs_files(build) {
                     Ok(Some(files)) => {
+                        if files.len() <= 10 {
+                            for file in &files {
+                                println!("formatting modified file {file}");
+                            }
+                        } else {
+                            println!("formatting only modified files");
```

Comment:
Can we add the count? It'll help with detecting if our modified detection is wrong (e.g. if the number is in thousands).